src/components/CheckboxGroup/README.md

```
@@ -0,0 +1,219 @@
+| required | Whether the checkbox is required | boolean | false |
+| readOnly | Whether the checkbox is read-only | boolean | false |
+| tabIndex | Tab index for keyboard navigation | number | - |
+| className | Additional CSS classes | string | - |
+| style | Inline styles | CSSProperties | - |
+| onChange | Callback when checkbox state changes | (e: CheckboxChangeEvent) => void | - |
+| onFocus | Callback when checkbox receives focus | (e: FocusEvent) => void | - |
+| onBlur | Callback when checkbox loses focus | (e: FocusEvent) => void | - |
+
+## Events
+
+### CheckboxChangeEvent
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)